README.md

```diff
@@ -31,10 +31,18 @@ GitLab CI cli for [changesets](https://github.com/atlassian/changesets) like its
 - `INPUT_TARGET_BRANCH` -> The merge request target branch. Defaults to current branch
 - `INPUT_CREATE_GITLAB_RELEASES` - A boolean value to indicate whether to create Gitlab releases after publish or not. Default true.
 
-### Outputs
+### Output
 
-- `PUBLISHED` - A boolean value to indicate whether a publishing is happened or not
-- `PUBLISHED_PACKAGES` - A JSON array to present the published packages. The format is `[{"name": "@xx/xx", "version": "1.2.0"}, {"name": "@xx/xy", "version": "0.8.9"}]`
+When publishing, a list of published packages is written to a file named `changesets-gitlab.output.json`.
```

Comment:
The output location should be mentioned and would be better to be configurable.

Reply:
I'll make it configurable. Another idea I had is to make the it configurable to have the output file at all, and default it to false. What do you think about that?

Reply:
Yes, opt-in configurable.